fil_actor_interface/src/builtin/known_cids.rs

```diff
@@ -111,27 +111,6 @@ mod tests {
 
     #[test]
     fn test_loading_static_value() -> Result<()> {
-        ensure!(crate::KNOWN_CIDS.actor.market.v8.contains(
```

Comment:
why did you remove it?

Reply:
In these we were ensuring correct values of only some versioned actor, didn't seem very useful